src/Form/Type/ListFilterType.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+
+namespace AlterPHP\EasyAdminExtensionBundle\Form\Type;
+
+
+use AlterPHP\EasyAdminExtensionBundle\Form\Transformer\Operator\GreaterThanOrEqualModelTransformer;
+use AlterPHP\EasyAdminExtensionBundle\Model\ListFilter;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\HiddenType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class ListFilterType extends AbstractType
+{
+    public static $operators = [
```

Comment:
`$operators` do not need to hold ORM version here, only the keys. I think this is QueryBuilder job to translate `equals` into `=` and so on.

Reply:
We should use constant to define the string for operators like :

```
    const OPERATOR_EQUALS = 'equals';
    const OPERATOR_GT = 'gt';
    const OPERATOR_GTE = 'gte';
    const OPERATOR_LT = 'lt';
    const OPERATOR_LTE = 'lte';

    private static $operators = [
        static::OPERATOR_EQUALS,
        static::OPERATOR_GT,
        static::OPERATOR_GTE,
        static::OPERATOR_LT,
        static::OPERATOR_LTE,
    ];
```

Reply:
I'm all for using constants.

The problem I see with delegating the translation to symbol in the QueryBuilder is that it couples the PostQueryBuilderSubscriber and ListFilterType, with no simple way of implementing any new  operator, modifying existing ones.

Not that we _need_ that, but my implementation allowed to handle all that logic in the FilterType

Reply:
There is a reason behind this willing : I developed a port of EasyAdmin for MongoDB documents, and EasyAdminExtension has a branch that is compatible with both entities and documents.

Operators like `=` or `>=` have no sense for Mongo query builder... That's why I prefer delegating DB query building to the builder.

I know this is more work to add filters, but decoupling the form from the DB operators makes sense.

Reply:
FYI https://github.com/alterphp/EasyAdminExtensionBundle/tree/feature/mongo-odm-compat/src/EventListener

Reply:
Ok it makes sense